Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -776,6 +794,22 @@ public void HandleUserConfirmationAsyncAlwaysSend()
             Mock.Get(ErrorLogHelper.Instance).Verify(instance => instance.InstanceRemoveStoredExceptionFile(_expectedManagedErrorLog.Id), Times.Never());
         }
 
+        [TestMethod]
+        public void HandleUserConfirmationAsyncAlwaysSendWhileDisabled()
+        {
+            ErrorLogHelper.Instance = GenerateMockErrorLogHelperWithPendingFile();
+
+            // Start Crashes.
+            Crashes.SetEnabledAsync(false).Wait();
+
+            Crashes.Instance.OnChannelGroupReady(_mockChannelGroup.Object, string.Empty);
+            Crashes.NotifyUserConfirmation(UserConfirmation.AlwaysSend);
+
+            // The alwaysSend value should never set because we are disabled
+            var alwaysSendValue = AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyAlwaysSend, false);
+            Assert.IsFalse(alwaysSendValue);
```

Comment:
Is there anything else we can verify other than always send flag?

Reply:
no as the collection is empty as its cleaned, so foreach loops are empty.
We want to see that it prints log of being inactive but we don't have asserts for logging.